src/pveclib/vec_f128_ppc.h

```diff
@@ -1184,13 +1377,162 @@ vec_signbitf128 (__binary128 f128)
 #if defined (_ARCH_PWR9) && defined (scalar_test_neg) && (__GNUC > 7)
   return (scalar_test_neg (f128);
 #else
-  vui32_t tmp, t128;
+  vui32_t tmp;
   const vui32_t signmask = CONST_VINT128_W(0x80000000, 0, 0, 0);
 
-  t128 = vec_xfer_bin128_2_vui32t (f128);
-  tmp = vec_and (t128, signmask);
+  tmp = vec_and_bin128_2_vui32t (f128, signmask);
   return vec_all_eq(tmp, signmask);
 #endif
 }
 
+/** \brief Scalar Insert Exponent Quad-Precision
+ *
+ *  Merge the sign (bit 0) and significand (bits 16:127) from sig
+ *  with the 15-bit exponent from exp (bits 49:63). The exponent is
+ *  moved to bits 1:15 of the final result.
+ *  The result is return as a Quad_precision floating point value.
+ *
+ *  \note This operation is equivalent to the POWER9 xsiexpqp
+ *  instruction and the built-in scalar_insert_exp. These require a
+ *  POWER9 enables compiler targeting -mcpu=power9 and are not
+ *  available for older compilers and POWER8 and earlier.
+ *  This operation provides this operation for all VSX enabled
```

Comment:
_nit_: "This operation provides this operation" is a bit awkward. Maybe "This function provides this operation"?
Also, s/VSX enabled/VSX-enabled/ (total nit)

Reply:
OK